smelly_python/code_smell.py

```diff
@@ -0,0 +1,28 @@
+from multiprocessing.dummy import Array
+
+
+class Code_Smell(object): 
```

Comment:
I think the naming convention is camelcase for classnames, so `CodeSmell`.
We should probably put pylint in the project already :sweat_smile:

Reply:
ack